packages/rlc-common/src/buildResponseTypes.ts

```diff
@@ -20,10 +20,12 @@ import {
   getResponseTypeName
 } from "./helpers/nameConstructors.js";
 
+let hasErrorResponse = false;
```

Comment:
Not sure there will be a bug in multi-client package.

When we have two clients but only one of them imports error response, but the other will not.

Reply:
already initate the value as false at line 28. each client will get a false at first.